### PR TITLE
Feat/404 page

### LINKED ISSUE #986: Redesign the 404 page
### What would you like to be added?

Currently, the 404 page leaves a lot to be desired.
404pageToday

Design a better, more informative 404 error page, which will give visitors options for moving forward from this "dead-end." And all that in a way—visually and verbally—that is more in line with MetaGame's brand.

### Why is this needed?

With MetaGame as complex as it is, we want to provide new visitors guidance with every step of their journey. At the same time, it would be prudent to provide a cohesive UX in all corners of the MetaGame Universe, and this is no exception.

### PR DESCRIPTION
**Pull request by firattale**
## Overview
Create a brand new 404 Page
**What features/fixes does this PR include?**

**Please provide the GitHub issue number**

Closes #986 

## Follow up Improvement Ideas

- [ ] please list any improvement/ideas

## Implementation

**Describe technical (nontrivial / non-obvious) parts of your code**

**Side effects**

## Assets

[Include screenshots/videos if it makes reviewing easier.]

### CODE REVIEW

**Comment by firattale** [comment]
https://frontend-pr-1013-mjhnbmqqna-uk.a.run.app/404
You can check the page here.

File: packages/web/pages/dashboard.tsx
Comment (dan13ram) [inline packages/web/pages/dashboard.tsx:57]: I think we should do this better than this. Currently the whole page loads and then we redirect. 
Instead we could show a loading screen until the user's logged in or connected.

Comment (firattale) [reply packages/web/pages/dashboard.tsx:57]: this line of code stayed from another PR, I can remove it here since this PR is about 404 page.

Comment (firattale) [reply packages/web/pages/dashboard.tsx]: I removed it @dan13ram

**Review by dan13ram — APPROVED** [review]
LGTM. We can merge. 
We should follow up with updating wherever we display 404 errors with a redirect to this page.
For example, when a player is not found in `/player/<username>`

**Comment by vidvidvid** [comment]
@dan13ram my comment about fixing a route was not addressed before you merged the PR. Can we make sure that we wait until the review comments are addressed next time?

**Comment by firattale** [comment]
> ent about fixing a route was not addressed before you merged the PR. Can we make sure that we wait until the review com

what was your comment @vidvidvid ?

**Comment by vidvidvid** [comment]
@firattale
Screenshot 2022-01-08 at 17 07 03

**Pull request merged**
